devoxx-2018-pl/cinema/src/main/resources/tweetwallConfig.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This file is missing the configuration of the StepEngine via key `stepEngine.steps`.
This results in
```
Exception in thread "JavaFX Application Thread" java.lang.NullPointerException: Configuration for 'stepEngine' does not exist
        at java.util.Objects.requireNonNull(Objects.java:228)
        at org.tweetwallfx.config.Configuration.getConfig(Configuration.java:197)
        at org.tweetwallfx.config.Configuration.getConfigTyped(Configuration.java:235)
        at org.tweetwallfx.controls.stepengine.StepIterator.create(StepIterator.java:67)
        at org.tweetwallfx.controls.stepengine.StepEngine.<init>(StepEngine.java:68)
        at org.tweetwallfx.controls.WordleSkin.prepareStepMachine(WordleSkin.java:277)
        at org.tweetwallfx.controls.WordleSkin.<init>(WordleSkin.java:190)
        at org.tweetwallfx.controls.Wordle.createDefaultSkin(Wordle.java:63)
        at javafx.scene.control.Control.impl_processCSS(Control.java:872)
        at javafx.scene.Parent.impl_processCSS(Parent.java:1280)
        at javafx.scene.Parent.impl_processCSS(Parent.java:1280)
        at javafx.scene.Node.processCSS(Node.java:9058)
        at javafx.scene.Scene.doCSSPass(Scene.java:545)
        at javafx.scene.Scene.access$3600(Scene.java:159)
        at javafx.scene.Scene$ScenePulseListener.pulse(Scene.java:2392)
        at com.sun.javafx.tk.Toolkit.lambda$runPulse$29(Toolkit.java:398)
        at java.security.AccessController.doPrivileged(Native Method)
        at com.sun.javafx.tk.Toolkit.runPulse(Toolkit.java:397)
        at com.sun.javafx.tk.Toolkit.firePulse(Toolkit.java:424)
        at com.sun.javafx.tk.quantum.QuantumToolkit.pulse(QuantumToolkit.java:518)
        at com.sun.javafx.tk.quantum.QuantumToolkit.pulse(QuantumToolkit.java:498)
        at com.sun.javafx.tk.quantum.QuantumToolkit.pulseFromQueue(QuantumToolkit.java:491)
        at com.sun.javafx.tk.quantum.QuantumToolkit.lambda$runToolkit$403(QuantumToolkit.java:319)
        at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
        at com.sun.glass.ui.mac.MacApplication._enterNestedEventLoopImpl(Native Method)
        at com.sun.glass.ui.mac.MacApplication._enterNestedEventLoop(MacApplication.java:109)
        at com.sun.glass.ui.Application.enterNestedEventLoop(Application.java:511)
        at com.sun.glass.ui.mac.MacView._enterFullscreen(Native Method)
        at com.sun.glass.ui.View.enterFullscreen(View.java:784)
        at com.sun.javafx.tk.quantum.WindowStage.applyFullScreen(WindowStage.java:647)
        at com.sun.javafx.tk.quantum.WindowStage.setFullScreen(WindowStage.java:737)
        at javafx.stage.Stage.setFullScreen(Stage.java:667)
        at org.tweetwallfx.generic.Main.start(Main.java:100)
        at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$161(LauncherImpl.java:863)
        at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$174(PlatformImpl.java:326)
        at com.sun.javafx.application.PlatformImpl.lambda$null$172(PlatformImpl.java:295)
        at java.security.AccessController.doPrivileged(Native Method)
        at com.sun.javafx.application.PlatformImpl.lambda$runLater$173(PlatformImpl.java:294)
        at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)

```